docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,130 @@
+We've used it in Material UI since v1 (back then, with JSS, and currently with Emotion in v5).
+However, there are key challenges we need to overcome to keep using this stack: improving performance and compatibility with React 19 features, including React Server Components.
+That's why, about a year ago, we started to work on Pigment CSS!
+We recently released an early alpha version, [introduced at React Conf 2024](https://mui.com/blog/introducing-pigment-css/), for which we're seeing [early positive adoption](https://npm-stat.com/charts.html?package=@stylexjs/stylex,@pigment-css/react) and [some community validation](https://x.com/JoshWComeau/status/1767224693575053692).
+
+This is just the beginning, though.
+There's so much to do to make Pigment CSS a reliable and mature option for the future of Material UI and for many developers looking for a next-generation CSS-in-JS styling engine.
+We're well-positioned to make this happen, so we're looking for a talented engineer to join the team and help us get further beyond!
+
+### Why this is interesting
+
+The core components that will depend on this library are widely used, you will receive a lot of feedback on your work.
+The community has high expectations, they will push you to do better, every day.
+
+Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
+
+## The role
+
+### Key responsibilities
+
+- **Build**. You will provide solutions for integrating the library with various bundlers and frameworks. You will build new features and improve the DX of the library.
+- **Nurture community contributions**. You will provide guidance and direction to unlock the contributions of the community. Your time will often be way better spent doing this than fixing the problems yourself.
+- **Shape the product**. You will be laser-focused on the end goal. It's not about solving technical challenges but about the problems solved for the users.
+- **Keep technical debt in check**. You will make sure we can keep shipping features at a reasonable pace, align the team on "one way" of doing things and make sure engineers follow the conventions.
+- **Be a solver**. You will dig deep into arbitrarily complex problems and find an appropriate path forward. Some focus on a given area for long periods, others bounce from hotspot to hotspot as guided by organizational leadership.
+
+### The best parts of this job
+
+- **You'll be at the cutting edge of application development** — working on one of the fastest-growing CSS-in-JS libraries on the market.
+- **You'll be part of an active, open, friendly community** of developers that are excited about building awesome applications.
+- **Your role will be key to making MUI the go-to UI solution** for building applications, websites, and design systems that does not have to trade performance for a great developer experience.
```

Comment:
```suggestion
- **Your role will be key to making MUI's libraries the go-to UI solutions** for building applications, websites, and design systems that do not have to trade performance for a great developer experience.
```

Same justification as the other — singular use of "MUI the go-to UI solution" sounds the same as  "Material UI the go-to UI solution".